torch/_inductor/triton_bundler.py

```diff
@@ -171,7 +185,7 @@ def collect(
                                 artifacts,
                             )
                         )
-                cls._entries = None
+                cls.end_compile()
```

Comment:
You don't need these anymore, although it is harmless

Reply:
i guess i do not need collect to end the compile as well anymore